editor/code_editor.cpp

```diff
@@ -757,6 +759,8 @@ FindReplaceBar::FindReplaceBar() {
 
 /*** CODE EDITOR ****/
 
+const float CodeTextEditor::ZOOM_FACTOR_PRESETS[7] = { 0.25f, 0.5f, 0.75f, 1.0f, 1.5f, 2.0f, 3.0f };
```

Comment:
This should allow declaring the variable only here and removing the declaration in the header:

```suggestion
static constexpr float CodeTextEditor::ZOOM_FACTOR_PRESETS[7] = { 0.25f, 0.5f, 0.75f, 1.0f, 1.5f, 2.0f, 3.0f };
```

Reply:
Did you mean to take out the `CodeTextEditor::` by any chance?

Reply:
It can probably be taken out too.

Reply:
It actually can't compile otherwise. 😅